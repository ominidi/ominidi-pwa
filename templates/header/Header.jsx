import React from 'react';
import { navItems } from '../../data/common.yml';
import Navigation from '../../components/navigation/Navigation';

const Header = () => (
  <header className="masthead">
    <div className="masthead__content">
      <div
        className="imaginery"
        style={{
          backgroundImage: 'url("https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/SantaCruz-CuevaManos-P2210651b.jpg/1200px-SantaCruz-CuevaManos-P2210651b.jpg")',
        }}
      />
      <h1 className="masthead__title">
        <a
          href="https://www.ominidi.org"
          title="Ominidi.org"
          className="color-white"
        >
          <img
            src="/static/assets/img/ominidi.org.png"
            title="Ominidi.org"
            alt="Ominidi.org"
            className="masthead__logo"
          />
        </a>
      </h1>
    </div>
    <Navigation navItems={navItems} />
  </header>
);

export default Header;
