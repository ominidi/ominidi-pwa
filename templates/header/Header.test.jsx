import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

jest.mock('next/router', () => ({
  pathname: '/',
}));

describe('Header', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
