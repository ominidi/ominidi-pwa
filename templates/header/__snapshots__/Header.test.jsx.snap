// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="masthead"
>
  <div
    className="masthead__content"
  >
    <div
      className="imaginery"
      style={
        Object {
          "backgroundImage": "url(\\"https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/SantaCruz-CuevaManos-P2210651b.jpg/1200px-SantaCruz-CuevaManos-P2210651b.jpg\\")",
        }
      }
    />
    <h1
      className="masthead__title"
    >
      <a
        className="color-white"
        href="https://www.ominidi.org"
        title="Ominidi.org"
      >
        <img
          alt="Ominidi.org"
          className="masthead__logo"
          src="/static/assets/img/ominidi.org.png"
          title="Ominidi.org"
        />
      </a>
    </h1>
  </div>
  <nav
    className="navigation"
  >
    <ul
      className="navigation__menu"
    >
      <li
        className="navigation__item"
      >
        <a
          className="navigation__link manifest"
          href="/"
          title="Manifest"
        >
          Manifest
        </a>
      </li>
      <li
        className="navigation__item"
      >
        <a
          className="navigation__link photos"
          href="/photos"
          title="Photos"
        >
          Photos
        </a>
      </li>
      <li
        className="navigation__item"
      >
        <a
          className="navigation__link attributions"
          href="/attributions"
          title="Attributions"
        >
          Attributions
        </a>
      </li>
      <li
        className="navigation__item"
      >
        <a
          className="navigation__link downloads"
          href="/downloads"
          title="Downloads"
        >
          Downloads
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
